srcpkgs/pipewire/files/pipewire-pulse/run

```diff
@@ -2,6 +2,7 @@
 # this service is experimental and most setups should start pipewire as a user,
 # for further information, please refer to the handbook
 ! [ -d /run/pulse ] && install -m 755 -g _pipewire -o _pipewire -d /run/pulse
+[ $(stat -c %a /run/pulse) -ne 755 ] && chmod 755 /run/pulse
```

Comment:
This can only happen if the directory is created manually, not by this service. In that case, I would say it's the user's responsibility and we shouldn't touch change it in the service.

Reply:
And the conditional is functionally useless - we could just run `chmod` unconditionally.